src/components/Login/Login.js

```diff
@@ -25,7 +30,78 @@ function Login() {
     setPass(e.target.value);
   };
   const submitHandler = () => {
-    console.log(username, pass);
+    setUsernam('');
+    setPass('');
+    if (logReg.status === false) {
+      fetch(
```

Comment:
do not do this in this file, please move it into separate file called api.ts and do it there